src/main/java/Duke.java

```diff
@@ -1,10 +1,142 @@
+import java.util.Scanner;
+
+
 public class Duke {
+
+    private static Task[] tasks = new Task[100];
+    private static int taskCounter =0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        startMsg();
+        Scanner input = new Scanner(System.in);
+
+        while (input.hasNextLine()) {
+
+            String command = input.nextLine().trim();
+            String[] doneTokens = command.split(" ");
+            if (command.equals("")) continue;
+
+            if(command.equals("bye")) {
```

Comment:
I think space bar is needed after the if and between the bracket for consistency throughout, some other instances of these in your code